apps/nowcasting-app/pages/_document.tsx

```diff
@@ -10,6 +10,7 @@ class MyDocument extends Document {
             rel="stylesheet"
           />
           <link rel="stylesheet" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css" />
+          <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"/>
```

Comment:
Not sure from looking at this URL what fonts and weights/options exactly it's requesting?

Reply:
me neither. I just copied and pasted the link from google fonts. i've typically worked with font awesome in the past. am open to other suggestions for fonts moving forward.